src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -28,7 +29,6 @@ import {
 import { GET_PATCH_TASKS } from "gql/queries/get-patch-tasks";
 import {
   useDisableTableSortersIfLoading,
-  usePollQuery,
```

Comment:
a feature of usePollQuery was to refetch data from the db whenever table params changed. since we rely on the apollo cache, we should only allow cached data to exist as long the pollInterval. This will ensure that any table page we navigate to will be fresh data from within the last minute (or whatever the pollInterval is set to)